test/fixtures.py

```diff
@@ -410,6 +410,8 @@ def start(self):
         jaas_conf = self.tmp_dir.join("kafka_server_jaas.conf")
         properties_template = self.test_resource("kafka.properties")
         jaas_conf_template = self.test_resource("kafka_server_jaas.conf")
+        self.ssl_dir = self.tmp_dir
+        gen_ssl_resources(self.ssl_dir.strpath) 
```

Comment:
I suppose this is a suitable replacement for `self.test_resource('ssl')`, but I'll need to double-check this to be sure.